ee/clickhouse/views/experiments.py

```diff
@@ -301,6 +301,9 @@ def create(self, validated_data: dict, *args: Any, **kwargs: Any) -> Experiment:
         feature_flag_serializer.is_valid(raise_exception=True)
         feature_flag = feature_flag_serializer.save()
 
+        if not validated_data.get("stats_config"):
```

Comment:
Just to be sure, all currently existing experiments have "stats_config" so this is only true for new ones?

Reply:
@andehen All existing statistics don't have a "stats_config" value at the moment. "stats_config" is only set for new experiments because this logic is only applied on the `create` function, though.

Reply:
I see, didn't notice it was in the create function, make sense 👍